coral/src/domain/helper/documentation-helper.ts

```diff
@@ -27,20 +27,41 @@ type MarkdownStringBrand = string & {
 
 type MarkdownString = Awaited<ReturnType<typeof createMarkdown>>;
 
+// very unique string to be able to identify documentation
+// transformation errors that otherwise can't be determined
+const documentationTransformError =
```

Comment:
Maybe using DOCUMENTATION_TRANSFORM_ERROR would be better for this extremely magical constant ^^

Reply:
Yes that makes sense! Visible const!